chia/wallet/cat_wallet/cat_wallet.py

```diff
@@ -297,31 +289,15 @@ async def get_confirmed_balance(self, record_list: Optional[Set[WalletCoinRecord
     async def get_unconfirmed_balance(self, unspent_records: Optional[Set[WalletCoinRecord]] = None) -> uint128:
         return await self.wallet_state_manager.get_unconfirmed_balance(self.id(), unspent_records)
 
+    @property
+    def cost_of_single_tx(self) -> int:
+        return 30000000  # Estimate
```

Comment:
we could consider memos here too?

Reply:
The behavior before didn't consider anything like that and memos could drive up the cost of a transaction, but it's very unlikely to include so many memos that we end up failing on fewer coins.